docs/cli.rst

```diff
@@ -0,0 +1,26 @@
+Command line interface
+======================
+
+In general, the only command that downstream repos will ever need to run is
+
+.. code-block:: bash
+
+    generate-bones all
+
+This updates all the templated files that need to be manually updated.  Note
+that there are other files that ``nengo-bones`` generates, but they are generated
+automatically by ``nengo-bones`` as needed, and will not need to be manually
+updated or committed to the downstream repo.
```

Comment:
Oh I missed the significance of this note. I read it and thought "yeah yeah" but didn't realize that this meant that "all" is not actually "all". The "all" setting is just the two root yml files for now. Maybe "all" could be renamed to "local", and then make it clear it should be run from the root directory to get the right defaults?

Reply:
Pushed a commit that improves this part of the documentation. Also changed "all" to the default command, as per offline discussion.